src/components/v5/common/CompletedAction/partials/PaymentBuilder/PaymentBuilder.tsx

```diff
@@ -88,9 +114,58 @@ const PaymentBuilder = ({ action }: PaymentBuilderProps) => {
     return uniqueTokens;
   }, []).length;
 
+  const hasPermissions = addressHasRoles({
+    address: user?.walletAddress || '',
+    colony,
+    requiredRoles: [ColonyRole.Arbitration],
+    requiredRolesDomains: [expenditure.nativeDomainId],
+  });
+  const showCancelOption =
+    expenditure?.status !== ExpenditureStatus.Cancelled ??
```

Comment:
Did you mean to use `&&`? `??` has no effect here, and cancel option shows for non-owners without permissions:

<img width="1023" alt="image" src="https://github.com/JoinColony/colonyCDapp/assets/112586815/cfb36e35-8dd4-4926-ab56-ff42da67a918">

Reply:
Yup, my mistake.